### PR TITLE
Add setup.py and instructions to use it

### PR DESCRIPTION
**Pull request by Yi-Tseng**
Also, fix the requirements.txt file since the jedi library 0.18 won't work with the latest ipython

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "p4runtime==1.3.0",
+    ],
+    author = "P4 API Working Group",
+    author_email = "p4-api@lists.p4.org",
+    classifiers = [
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+    ],
+    description = "The P4 Runtime shell",
```
Comment (antoninbas) [inline setup.py:25]:
```suggestion
    description = "The P4Runtime shell",
```

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
-ipaddr
```
Comment (antoninbas) [inline requirements.txt:1]: do we still need that file with your changes?

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,9 @@
 *~
 *.pyc
 .coverage
+venv
+build
+dist
+*.egg-info
+.vscode
+
```
Comment (antoninbas) [inline .gitignore:9]: extra empty line?

File: requirements.txt (hunk 2 above)
Comment (Yi-Tseng) [reply requirements.txt:1]: right, actually we can remove it now I think

**Review by antoninbas — APPROVED** [review]
LGTM, thanks for the contribution!

**Pull request merged**